config/karma.config.js

```diff
@@ -3,11 +3,12 @@
 
 import webpackConfig from './webpack.config';
 
-var coverageReporters = [{
+const coverageReporters = [{
   type: 'text-summary',
 }];
 
-var newWebpackConfig = Object.assign({}, webpackConfig, {
+// eslint-disable-next-line prefer-const
+let newWebpackConfig = Object.assign({}, webpackConfig, {
```

Comment:
I'm not sure eslint is getting it "wrong" here. `const` just means the value can't be reassigned, not that it can't be mutated. I think `const` would be needed for immutability but it doesn't enforce immutability (the type also needs to be immutable).

I think this should still be `const` even though it gets mutated. I'm fine with it as is though, it can get changed if someone feels like changing it.

These articles have some mildly interesting points: I did a search and found an article called *[ES6 `const` is not about immutability](https://mathiasbynens.be/notes/es6-const)* which goes into that and someone posted another article in the comments there about [why you *shouldn't* use `const` by default](http://blog.getify.com/constantly-confusing-const/#premature-const), which is interesting but I'm not entirely convinced.